### ISSUE TITLE
RCTBridgeModule 'file now found' issues

### ISSUE THREAD

**Issue by abdrmdn** [KjellConnelly/react-native-rate#34]
#ISSUE 
I tried the 3 ways of linking the lib to ios but this doesnt seem to be working fine (beside andorid which works just fine).
Issue happening is shown below:
```
In file included from /Users/studocu/development/app/node_modules/react-native-rate/ios/RNRate.h:3:
../../react-native/React/Base/RCTBridgeModule.h:10:9: fatal error: 'React/RCTDefines.h' file not found

#import <React/RCTDefines.h>
        ^~~~~~~~~~~~~~~~~~~~

1 error generated.



** BUILD FAILED **
```

Some people suggesting changes in React-native folders and so on. Not sure how to make this work though!?

FYI:
react-native-cli: 2.0.1
react-native: 0.58.6
Xcode 10.2
react-native-rate: ^1.1.6

**Comment by abdrmdn** [comment]
And I found that since react-native 0.40 all libraries of react will be called with "<>" instead.
Check :
https://github.com/facebook/react-native/releases/tag/v0.40.0

to be continueed...

**Comment by allhaze** [comment]
Same issue :(

**Comment by abdrmdn** [comment]
PR above might help Fixing part of this.

**Comment by Crizzooo** [comment]
My react native is 0.58.9, react 16.8.3, and react-native-rate 1.1.17.

Same issue regarding `RTCDefines.h` after running link, adding `pod 'RNRate', :path => '../node_modules/react-native-rate/ios'` to PodFile, and pod install. 

Any ideas on how to fix this?

**Comment by Crizzooo** [comment]
Or actually, employing above fix to use `<>` instead of `""`appears to build with `react-native run-ios` although it fails in Xcode.

But `react-native run-ios` then errors out with the below

`
..../node_modules/react-native-rate/ios/RNRate.m:19:14: note: enclose 'SKStoreReviewController' in an @available check to silence this warning
        if ([SKStoreReviewController class]) {
             ^~~~~~~~~~~~~~~~~~~~~~~

info ...../node_modules/react-native-rate/ios/RNRate.m:22:18: warning: 'SKStoreReviewController' is only available on iOS 10.3 or newer [-Wunguarded-availability]

info                 [SKStoreReviewController requestReview];
`

**Comment by abdrmdn** [comment]
Its working for me now
The native side didn’t have any code to run when using the react-native-rate package from JS, now it does. Just don’t forget to run cd ios && pod install before testing.
